tests/test_e2e_60_of_multi_table.py

```diff
@@ -0,0 +1,430 @@
+                    "table_id": 2,
+                    "description": "Third table for of_lldp",
+                    "napps_table_groups": {
+                        "of_lldp": ["base"]
+                    },
+                    "table_miss_flow": {
+                        "priority": 0,
+                        "instructions": [{
+                            "instruction_type": "goto_table",
+                            "table_id": 3
+                        }]
+                    },
+                },
+                {
+                    "table_id": 3,
+                    "description": "Fourth table for mef_eline evpl",
+                    "napps_table_groups": {
+                        "mef_eline": ["evpl"],
+                    },
+                    "table_miss_flow": {
+                        "priority": 0,
+                        "instructions": [{
+                            "instruction_type": "goto_table",
+                            "table_id": 4
+                        }]
+                    },
+                },
+                {
+                    "table_id": 4,
+                    "description": "Fifth table for mef_eline epl",
+                    "napps_table_groups": {
+                        "mef_eline": ["epl"]
+                    },
+                },
+            ]
+        }
+        evc = {
+            "name": "evc01",
+            "enabled": True,
+            "dynamic_backup_path": True,
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:01:1",
+                "tag": {"tag_type": 1, "value": 100}
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:01:2"
+            }
+        }
+
+        # Add circuit
+        api_url = f"{KYTOS_API}/mef_eline/v2/evc/"
+        response = requests.post(api_url, json=evc)
+        assert response.status_code == 201, response.text
+        data = response.json()
+        assert 'circuit_id' in data
+        time.sleep(10)
+
+        # Add pipeline
+        api_url = f"{KYTOS_API}{OF_MULTI_TABLE_API}"
+        response = requests.post(api_url, json=pipeline)
+        data = response.json()
+        assert response.status_code == 201
+        assert 'id' in data
+
+        # Enabled pipeline
+        api_url = f"{KYTOS_API}{OF_MULTI_TABLE_API}/{data['id']}/enable"
+        response = requests.post(api_url)
+        assert response.status_code == 200
+        time.sleep(10)
+
+        # Get pipeline dictionary
+        api_url = f"{KYTOS_API}{OF_MULTI_TABLE_API}/{data['id']}"
+        response = requests.get(api_url)
+        pipeline_data = response.json()
+        assert response.status_code == 200
+        assert pipeline_data["status"] == "enabled"
+
+        # Assert installed flows
+        s1 = self.net.net.get('s1')
+        flows_s1 = s1.dpctl('dump-flows').splitlines()
+        assert "table=0" in flows_s1[0]
+        assert 'priority=0 actions=resubmit(,1)' in flows_s1[0]
```

Comment:
@Alopalao, regarding `actions=resubmit(,x)` to be safe, could you also add another logical or `assert` including also the same clause except with `goto_table:x` too? It looks coherent here, but to avoid this test to give a false positive if a different version of `OvS` start using one output or the other.

Reply:
Also, if you could assert the number of expected flows upfront that'd be great. If a new NApp that has a higher priority or additional flows on table 0 it'll break the sequence her to avoid confusion and facilitate the analysis when that ever happens. Also, if you could group by tables and perform the asserts accordingly based on also the number of expected flows your `asserts` would be more robust and easier to maintain in the long run.